bot/cogs/information.py

```diff
@@ -192,6 +204,23 @@ def __init__(self, bot: Bot):
 
         await ctx.send(embed=embed)
 
+    @user_info.error
+    async def user_info_command_error(self, ctx: Context, error: CommandError):
+        embed = Embed(colour=Colour.red())
+
+        if isinstance(error, BadArgument):
+            embed.title = random.choice(NEGATIVE_REPLIES)
+            embed.description = "You do not have permission to use this command on users other than yourself."
+            await ctx.send(embed=embed)
```

Comment:
If the `user` does not convert to a member of the guild, we get a `BadArgument`, resulting in this confusing error message:

![image](https://user-images.githubusercontent.com/22679924/51083837-c8658700-1720-11e9-814e-1dc40acc4171.png)

I propose we somehow use the inner description of the exception instead.